### DISCUSSION TITLE
thanhle.blog/en/blog/ki-nang-vi-mo-va-vi-mo-cho-dev

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#42] [Ideas]
# thanhle.blog/en/blog/ki-nang-vi-mo-va-vi-mo-cho-dev

Kĩ năng vĩ mô và vi mô cho dev để chọn career tốt hơn

https://thanhle.blog/en/blog/ki-nang-vi-mo-va-vi-mo-cho-dev

**Comment by binhnguyen2501** [comment]
😍
